### ISSUE TITLE
Datatable exporter to XLS breaks if header contains Inputtext

### ISSUE THREAD

**Issue by SegfaultSorcerer** [primefaces/primefaces#1644]
Hi,
i want to use the Datatable Exporter in my application. I have a usual datatable with the globalsearch field  inside the header facet. I use PF 6.0. If i click on the export button the following happens: 

ExcelExport.java (Primefaces lib)

```
Workbook wb = createWorkBook();
String sheetName = getSheetName(context, table);
if(sheetName == null) {
    sheetName = table.getId();
}
```

The method getSheetName trys to figure out the title of the datatable by iterating through the header facet components. The method `ComponentUtils.getValueToRender(context, child);` grabs for the value of the components.  But if there is an Inputtext field  which is empty it takes the empty String to create a new sheet with the apache poi workbook api. This does not work with poi version 3.14 because the sheet name needs to contain at least 1 char. 

Maybe a quickfix would be to add 

```
if(sheetName == null || sheetName.equals("")) {
    sheetName = table.getId();
}
```

**Comment by mertsincan** [comment]
I think this issue is fixed for 6.0.1. 
Issue: #1552

**Comment by SegfaultSorcerer** [comment]
Oh, yes. I searched for an issue like this but did not find this. 

Thank you :+1:
